src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -53,6 +53,14 @@ public function getConfigTreeBuilder()
                     return $v;
                 })
             ->end()
+            ->beforeNormalization()
```

Comment:
I think this should rather be done on ``validate``. Otherwise, you will create issues if 1 file uses the old syntax, because you will inject the new syntax in all other config files being merged